addon/components/polaris-action-list.js

```diff
@@ -44,6 +44,8 @@ export default Component.extend({
    */
   onActionAnyItem() {},
 
+  hasMultipleSections: gt('finalSections.length', 1).readOnly(),
```

Comment:
This is an internal property so I'd prefer it to not end up up here with the public properties... 😬

Reply:
It hasn't

Reply:
It's still above the comment which marks the start of the "internal properties" section of the file. Can we remove that comment and be sure to explicitly mark all properties as `@public` of `@private` please?